frontend/app/components/CardBack.tsx

```diff
@@ -256,6 +274,23 @@ const ExamplesContent = ({
             />
           ) : null,
         )}
+
+        {examples.length > 1 && !showAllExamples && firstExampleComplete && (
+          <TouchableOpacity
+            onPress={handleShowMoreExamples}
+            style={[
+              styles.showMoreButton,
+              { backgroundColor: colors.cardBackBackground },
+            ]}
+            activeOpacity={0.8}
+          >
+            <View style={styles.showMoreContent}>
+              <Ionicons name="add-circle-outline" size={20} color="#FFD700" />
+              <Text style={styles.showMoreText}>Show more examples</Text>
+              <Ionicons name="chevron-down" size={16} color="#FFD700" />
```

Comment:
Just wondering if we need both icons here, gut feeling is just text should suffice or not? Let's have a look on Monday :)